src/mess/drivers/hp64k.c

```diff
@@ -349,6 +385,26 @@ void hp64k_state::video_start()
 		m_chargen = memregion("chargen")->base();
 }
 
+// Divisors of K1135 baud rate generator
```

Comment:
On 08/17/2015 05:49 PM, Jonathan Gevaryahu wrote:
>
> In src/mess/drivers/hp64k.c
> <https://github.com/mamedev/mame/pull/286#discussion_r37202087>:
>
> > @@ -349,6 +385,26 @@ void hp64k_state::video_start()
> >  		m_chargen = memregion("chargen")->base();
> >  }
> >  
> > +// Divisors of K1135 baud rate generator
>
> I think we already have a device for this type of baud divider, in
> machine/com8116.h
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamedev/mame/pull/286/files#r37202087>.
>
Thanks for your suggestion, I completely missed that out. When time
allows I'll refactor the code to use com8116. At a closer look, this
component looks like a close relative of K1135 that HP 64000 used.

-- F.Ulivi